### PR TITLE
Provide justification information for mkl-threads

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: _j/mkl_threads.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: Consider adjusting ``OMP_NUM_THREADS`` environment variable for containerized deployments
+---
+
+Consider adjusting ``OMP_NUM_THREADS`` environment variable for containerized
+deployments, one or more libraries use Intel's MKL that does not detect
+correctly one or more libraries use Intel's MKL that does not detect correctly
```
Comment (pacospace) [inline _j/mkl_threads.md:7]: `one or more libraries use Intel's MKL that does not detect correctly` it is repeated

File: _j/mkl_threads.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+resource allocation in the cluster resource allocation in the cluster.
```
Comment (pacospace) [inline _j/mkl_threads.md:8]: `resource allocation in the cluster` repeated as well

File: _j/mkl_threads.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+ * MKLThreadsWrap
+
+## Recommendation types
+
+ * latest
+ * performance
+ * secure
+ * stable
+ * testing
```
Comment (pacospace) [inline _j/mkl_threads.md:41]: all reccomendations? Don't we have docs to be linked for each typo of recommendation?

Comment (fridex) [reply _j/mkl_threads.md:41]: :)

**Comment by fridex** [comment]
All the comments addressed. I'll merge this one not to have it stalled - a follow up PR will adjust the links.

**Pull request merged**